### ISSUE TITLE
Unable to load HEIC file since 1.14.0

### ISSUE THREAD

**Issue by julianfairfax** [strukturag/libheif#742]
Hi,

I am running the GNOME Image Viewer app on Arch, and I am unable to view HEIC images in it starting from https://github.com/flathub/org.gnome.eog/commit/16c633d413d71c6e23368a0aed61ede4ed10a340, which switched to version 1.14.0 of libheif. Downgrading fixes this, but is obviously not a solution.

Error log:
```
** (eog:2): WARNING **: 11:49:01.937: Cannot read plugin directory.

(eog:2): GdkPixbuf-CRITICAL **: 11:49:01.938: gdk_pixbuf_animation_is_static_image: assertion 'GDK_IS_PIXBUF_ANIMATION (animation)' failed

(eog:2): GdkPixbuf-CRITICAL **: 11:49:01.938: gdk_pixbuf_animation_get_iter: assertion 'GDK_IS_PIXBUF_ANIMATION (animation)' failed

(eog:2): GdkPixbuf-CRITICAL **: 11:49:01.938: gdk_pixbuf_animation_iter_get_pixb
```

Ref: https://github.com/flathub/org.gnome.eog/issues/73

**Comment by TingPing** [comment]
I bisected this and 5fb3a74ec1dee4624c03a66eaf63b1fa3fee13c0 is the first bad commit.

**Comment by TingPing** [comment]
This is obviously not the solution but this makes libheif work:

```diff
diff --git a/libheif/heif_init.cc b/libheif/heif_init.cc
index 6f0df64..d2981b9 100644
--- a/libheif/heif_init.cc
+++ b/libheif/heif_init.cc
@@ -60,7 +60,7 @@ struct heif_error heif_init(struct heif_init_params*)
 
   heif_library_initialization_count++;
 
-  if (heif_library_initialization_count == 1) {
+  if (0) {
 
     // --- initialize builtin plugins
 ```

**Comment by TingPing** [comment]
I fixed this for cmake in #744, however I have not looked at the autotools build system and fixed it.

**Comment by farindk** [comment]
@TingPing The autotools build system is deprecated and will be removed anyway.
